docs/modules/ROOT/pages/explanations/slos.adoc

```diff
@@ -0,0 +1,40 @@
+= Service Level Objectives
+
+APPUiO Managed OpenShift 4 comes with a collection of https://sre.google/sre-book/service-level-objectives/[service level objectives (SLOs)].
+This document defines and explains these SLOs.
+An APPUiO Managed cluster should meet these objectives to provide the expected service level to our customers.
+
+We use the SLOs and https://sre.google/workbook/alerting-on-slos/#6-multiwindow-multi-burn-rate-alerts[Multiwindow, Mulit-Brun-Rate Alerts] as the basis of our on-call alerting.
+
+IMPORTANT: These are internal sevice level *objectives*, not service level *agreements*.
+We don't guarantee to meet these objectives at all times.
+
+== Storage
+
+Persistent storage is a key component of a feature complete Kubernetes cluster.
+Any storage issues directly impacts the service level for users.
+
+=== CSI Operations
+
+****
+*99.5% of all CSI operations complete successfully*
```

Comment:
I pulled this number out of my arse. I defined it here https://github.com/appuio/component-openshift4-slos/blob/master/class/defaults.yml#L30. 

I have no Idea what would be a good objective. I propose we revisit all of these SLOs once he actually have them